front/app/modules/commercial/content_builder/admin/components/ContentBuilderToggle/ContentBuilderToggle.test.tsx

```diff
@@ -51,8 +68,13 @@ describe('ContentBuilderToggle', () => {
     expect(screen.queryByText('LinkText')).not.toBeInTheDocument();
     toggle.click();
     expect(screen.queryByText('LinkText')).toBeInTheDocument();
+    const mockLayoutData = { enabled: true } as IContentBuilderLayoutObject;
```

Comment:
This is not needed, you can directly add it to the expect:
```
expect(addContentBuilderLayout).toHaveBeenCalledWith(
      { projectId: 'projectId', code: undefined },
      { enabled: true }
    );
```

Also, avoid using `as` if you can :)